src/adhocracy_core/adhocracy_core/content/__init__.py

```diff
@@ -307,7 +308,11 @@ def get_workflow(self, context: object) -> IWorkflow:
 
     def can_add_anonymized(self, context: object, request: Request) -> bool:
         """Check if children can be created/added anonymized to `context`."""
-        has_sheet = IAllowAddAnonymized.providedBy(context)
+        if IItem.providedBy(context):
+            # if item was anonymized you can also add versions anonymized
+            has_sheet = _is_anonymized_and_has_permission(context, request)
```

Comment:
Shouldn't the check be for ItemVersion and the call for the parent item?

has_sheet is misleading at that point and the permissions check is also done two times.